src/main/scala/mesosphere/marathon/state/RootGroup.scala

```diff
@@ -408,12 +408,12 @@ class RootGroup(
       Group(
```

Comment:
please update the comment for this method, it still says `Timestamp(0)`

btw. why do we have this method here anyway? It seems to be used in tests only...